BLSQ/iaso
# IA-2760: Update default Projects (mobile app IDs)

Related JIRA tickets : [IA-2760](https://bluesquare.atlassian.net/browse/IA-2760)

## Self proofreading checklist



## Changes

Created 3 additional projects: **Planning, Georegistry/Géoregistre and Children vaccination/Vaccination des enfants** by adding an argument in the script launcher to specify the creation of 3 additionnals projects!

## How to test

From setuper folder, launch setuper via `python3 setuper.py --additionnal_projects` to generate test data. 
Then login with the created account and open projects page: [http://localhost:8081/dashboard/settings/projects/](http://localhost:8081/dashboard/settings/projects/).

You should see 3 additional projects: **Planning, Georegistry/Géoregistre and Children vaccination/Vaccination des enfants**

From the forms page, check for the forms: 
  - Equipment/Pop/Social mob./Microplans: you should see 2 additionals projects(Planning and Georegistry/Géoregistre)
  - Child - Registration and Child - Follow up: 1 additional project(Children vaccination/Vaccination des enfants)


## Print screen / video


https://github.com/BLSQ/iaso/assets/6383219/52c44ffc-b835-410d-a3d8-bc0e428f7101


[IA-2760]: https://bluesquare.atlassian.net/browse/IA-2760?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
